src/v2/Components/__stories__/Molecules.story.tsx

```diff
@@ -0,0 +1,200 @@
+import {
+  Box,
+  Text,
+  GridColumns,
+  Column,
+  Button,
+  Spacer,
+  ProgressDots,
+} from "@artsy/palette"
+import { themeGet } from "@styled-system/theme-get"
+import React from "react"
+import styled from "styled-components"
+import { FullBleed } from "../FullBleed"
+
+export default {
+  title: "Molecules/Kitchen Sink",
+  parameters: {
+    layout: "fullscreen",
+  },
+}
+
+const AppContainer = styled(Box)`
+  max-width: ${themeGet("breakpoints.lg")};
+`
+
+AppContainer.defaultProps = {
+  mx: "auto",
+  px: [2, 4],
+}
+
+const Page: React.FC = ({ children }) => {
+  return <AppContainer>{children}</AppContainer>
+}
+
+const MetadataExample = () => {
+  return (
+    <>
+      {/* TODO: Mixin a textTransform prop to text */}
+      <Text variant="xs" style={{ textTransform: "uppercase" }} mb={1}>
```

Comment:
It looks like we'll be doing a lot of all caps small type treatments so just mixing in a textTransform prop is a good idea.

Reply:
There's been some mumbling back and forth around the upper-case text treatment, but not sure if anything has changed. Might as well mix it in and ping Jonathan to see if there's any movement there. (I'm assuming there's not and we just push ahead.)